### ISSUE TITLE
Kernel landing page

### ISSUE THREAD

**Issue by thesachinmittal** [gitcoinco/creative#364]
Kernel lives at https://kernel.community/

But we require a newer version of the home page of https://kernel.community/.

Driver: @GuistF 
Approver: @nglglhtr @andytudhope

**Comment by nglglhtr** [comment]
- what should the newer page look like? how different?
- I do see kernel-v2 progress going on at https://github.com/kernel-community/kernel-v2/, I think it would make most sense to have one there itself? 🤔

**Comment by andytudhope** [comment]
Yeah, I would also prefer to have discussions related to the home page in the actual repo where I am working on it. Nevertheless, this is what it currently looks like. I think it's nice, but would be happy to discuss it more in depth with a designer once the gratitude page has been done (it is definitely the first priority in my mind):

![Screenshot from 2021-04-13 09-42-51](https://user-images.githubusercontent.com/13001517/114515616-d62a4f80-9c3c-11eb-993b-284bb7d6565b.png)
